.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           path: collect_app/build/reports
           destination: reports
 
-  test_unit:
+  test_modules:
```

Comment:
This new job allows us to run all the submodule tests in parallel to the app tests and the quality checks which as we move more things out to submodules will hopefully keep our test CI time under control